src/components/shared/Sidebar.jsx

```diff
@@ -0,0 +1,15 @@
+import classnames from 'classnames'
+import React, { Children } from 'react'
+
+import classNames from './Sidebar.scss'
+
+export const Sidebar = ({ children, scrollable, width, maxWidth = '50%', flex = 6, ...props }) => {
```

Comment:
Is there a strong reason to have a shared `Sidebar` component? I think we could simplify by declaring this in the place we use it, if it's only one place, and designing it to be specific rather than generalizing it.